06-Modelling-Rigid-Bodies/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<p>Why are you shifting all particles by 1.3 in all directions?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>So that the rigid bodies are whole in the renders and do not have some particles wrapped around the periodic boundary conditions.</p><p> </p><p>It should be spacing/2, not 1.3.</p>